logstash-core/spec/logstash/agent_spec.rb

```diff
@@ -425,12 +473,13 @@ class DummyOutput2 < LogStash::Outputs::DroppingDummyOutput; end
       Thread.abort_on_exception = true
 
       @t = Thread.new do
```

Comment:
actually, I fixed the pipeline thread with a `let` but kept the `@abort_on_exception` thing.